models/delete-models.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+exports.removeComment = (comment_id) => {
+  return db.query(`DELETE FROM comments WHERE comment_id = $1`, [comment_id]);
+};
+
+exports.checkCommentExists = (comment_id) => {
+  return db
+    .query(`SELECT * FROM comments WHERE comment_id = $1`, [comment_id])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
+      return rows[0];
+    });
+};
```

Comment:
This does work, but you can do this in a single query, what could you add to your delete query to give you some info back to check what has been deleted?